processes/third_party_vuln_submit_form_hacker1.md

```diff
@@ -0,0 +1,59 @@
+> NOTE! Thanks for submitting a report! Please replace *all* the [square] sections below with the pertinent details. Remember, the more detail you provide, the easier it is for us to triage and respond quickly, so be sure to take your time filling out the report!
+
+I would like to report [VULNERABILITY] in [MODULE]
+It allows [DESCRIBE THE IMPACT OF THE VULNERABILITY - E.G READ ARBITRARY FILES, READ DATA FROM DATABASE ETC]
```

Comment:
There is useless duplication between this header, «# Module» section, «## Vulnerability Description» section, and HackerOne «Impact» field (which is not shown in this template, but is present as a separate field on HackerOne).

Perhpaps this could be simplified somehow.

Reply:
I see what you mean, and the impact field is also required so there's no way to avoid it anyway, right?

What I would want to do is remove this part and add it in the impact field but we don't have access to customize that so I find this intro useful in providing a guideline.